release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1699,6 +1711,26 @@ In the following tables, features are marked with the following statuses:
 |Not Available
 |Technology Preview
 
+|MT2892 Family [ConnectX-6 Dx] OvS Hardware Offload and SR-IOV support
```

Comment:
We can have separate rows  "<Name of NIC> SR-IOV support" (This is distinctly different from "OvS Hardware Offload and SR-IOV support") which follows the history of https://docs.openshift.com/container-platform/4.11/networking/hardware_networks/about-sriov.html#supported-devices_about-sriov since 4.10

Reply:
Looks good now.